object/basic.go

```diff
@@ -97,6 +97,7 @@ func GetHighestOnlineNum() int {
 }
 
 func UpdateHighestOnlineNum(num int) bool {
+	highestOnlineNum = num
```

Comment:
I think the reason for the incorrect maximum online number is that it is not updated to the memory in real-time.

Reply:
It just updates this file in databse, but not to update in memory. If maxonlineNumber updates, we won't get the latest right now, unless we reset the system and read from the database.

Reply:
great found! Let's see the result in production.